common/src/main/kotlin/org/jetbrains/kotlinconf/presentation/CodeVerificationPresenter.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.kotlinconf.presentation
+
+import org.jetbrains.kotlinconf.api.KotlinConfApi
+import org.jetbrains.kotlinconf.launchAndCatch
+import kotlin.coroutines.CoroutineContext
+
+class CodeVerificationPresenter(
+        private val uiContext: CoroutineContext,
+        private val view: CodeVerificationView,
+        private val repository: DataRepository
+) {
+
+    fun onCreate() {
+        launchAndCatch(uiContext, view::showError) {
+            val api = KotlinConfApi()
```

Comment:
Could we avoid creating the blank API?

Reply:
Yes, I used it only for testing purposes.